src/bridge-deployer/src/bridge_deployer.rs

```diff
@@ -55,7 +55,7 @@ impl BridgeDeployer {
 
         let canister_id = self.client.client().canister_id;
         let management_canister = ManagementCanister::create(&self.agent);
-        let arg = config.init_raw_arg(network)?;
+        let arg = config.init_raw_arg(self.agent.get_principal().unwrap(), network)?;
```

Comment:
Replace `unwrap()` with `?`

Reply:
This function returns `String` error which cannot be cleanly converted to `anyhow::Error`. But this code should never fail with valid identity, so I just replaced `unwrap` with `expect`.